### PR TITLE
feat: Update Phishing Controller to version 6.0.0

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**
Update Phishing Controller to version 6.0.0

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**
http://recordit.co/TZ9KbX6rFq

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/util/test/initial-background-state.json [hunk 1]
```diff
@@ -75,17 +75,7 @@
...
-        "allowlist": [],
-        "blocklist": [],
-        "fuzzylist": [],
-        "tolerance": 2,
-        "version": 2,
-        "name": "MetaMask",
-        "lastUpdated": 0
-      }
-    ],
+    "phishingLists": [],
```
Comment (MarioAslau) [inline app/util/test/initial-background-state.json:78]: This is really cool! I've actually had to deactivate this on my branch haha

**Review by MarioAslau — APPROVED** [review]
🎉🎉🎉

**Review by Cal-L — APPROVED** [review]
LGTM

**Comment by tommasini** [comment]
@SocketSecurity ignore-all
Ignoring all since there is no new dependency added to the phishing controller on this update

**Pull request merged**